app/lib/pages/receive_page.dart

```diff
@@ -41,6 +43,14 @@ class _ReceivePageState extends State<ReceivePage> with Refena {
     WidgetsBinding.instance.addPostFrameCallback((_) async => _init());
   }
 
+  @override
+  void dispose() {
+    super.dispose();
+    if (Platform.isWindows) {
```

Comment:
Extract platform-specific handling to utils (here you can do a noop for Linux (?) and macOS)